src/transformers/models/fuyu/modeling_fuyu.py

```diff
@@ -261,9 +261,8 @@ def forward(
         if inputs_embeds is None:
             inputs_embeds = self.language_model.get_input_embeddings()(input_ids)
             if image_patches is not None and past_key_values is None:
-                patch_embeddings = self.vision_embed_tokens(
-                    image_patches.to(self.vision_embed_tokens.weight.dtype)
-                ).to(inputs_embeds.device)
+                patch_embeddings = self.vision_embed_tokens(image_patches.to(self.vision_embed_tokens.weight.dtype))
```

Comment:
is there another way to do this?

Reply:
I need those two embeddings to be on the same device. I can't use `_no_split_modules` to force those two modules to be on the same device. Moreover, those two modules dont share weights. So there is no way to make sure that they are on the same device if we don't move them.

Reply:
I see. Fine with me!